app/contestants/page.tsx

```diff
@@ -4,14 +4,14 @@ const wikiUrl = "https://en.wikipedia.org/wiki/The_Amazing_Race_35"
 
 async function getData() {
 
-    const response = await fetch(wikiUrl)
+    const response = await fetch(wikiUrl, {next: { revalidate: 3600 }})
     const responseText = await response.text()
     const doc = new JSDOM(responseText)
     var domQuery = doc.window.document.querySelectorAll(".vcard .fn")
 
     // Build data model
     const final = []
-    var team = ""
+    var team = {teamName: "", isParticipating: true}
```

Comment:
Would it be better to set the default value here for `isParticipating` to `false`?

Reply:
Maybe, my thought was that every team starts the game participating so in a way, that is the default. But I am open to a counter.